test/puppetlabs/puppetdb/http/events_test.clj

```diff
@@ -349,7 +349,7 @@
        :new_value nil
        :containing_class "Foo"
        :report_receive_time "2014-04-16T12:44:40.978Z"
-       :report "99ec099bed6dfb9bff2c7df7828270e95f590147"
+       :report "7d0cfa08901e1e1d80cf2f2f814d356d0e457e09"
```

Comment:
OK, right, so at least as a practical matter, I added some code to print the json in generate string whenever including the pdb custom date format makes a difference, and saw this:

```
2011-01-01T15:00:01Z
2011-01-01T15:00:01.000Z
```
I was assuming that's what changed the hash, though I haven't investigated the report hashing explicitly.  But I can if you like.